site/published-version.json

```diff
@@ -1,5 +1,5 @@
 {
     "version":"0.0.1",
     "message":"New version is available",
-    "url":"https://github.com/HewlettPackard/hpecli/releases/download/0.0.1/latest_0.0.1_windows_amd64.exe"
+    "url":"https://github.com/HewlettPackard/hpecli/releases/download/0.0.2/latest_0.0.2_{{$GOOS}}_amd64.exe"
```

Comment:
do we want to start with 0.0.2?
Mac & Linux will have different file names .. not .exe

Reply:
Ha yes. I forgot about the .exe. I'll fix

Reply:
Added parameter in URL for .EXE or not